operators/pkg/tenant-controller/tenant_controller.go

```diff
@@ -208,7 +210,7 @@ func (r *TenantReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 	}
 
 	// no retrigErr, need to normal reconcile later, so need to create random number and exit
-	nextRequeSeconds, err := randomRange(3600, 7200) // need to use seconds value for interval 1h-2h to have resolution to the second
+	nextRequeSeconds, err := randomRange(3600*6, 3600*8) // need to use seconds value for interval 6h-8h to have resolution to the second
```

Comment:
```suggestion
	nextRequeSeconds, err := randomRange(3600*4, 3600*8) // need to use seconds value for interval 4h-8h to have resolution to the second
```

Allowing a wider interval, the concentration effect should decrease (since at restart all tenants are processed at the same time)